From 6dab6527a7d7c943bc9a2a4ae2355e876406f5b7 Mon Sep 17 00:00:00 2001
From: Kyle Gospodnetich <me@kylegospodneti.ch>
Date: Fri, 7 Aug 2026 16:40:04 -0700
Subject: [PATCH] fix: Add DRM leasing options to help for gamescope_has_option
 checks

---
 src/main.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/main.cpp b/src/main.cpp
index 6b574572cd..0e02e2977a 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -216,6 +216,8 @@ const char usage[] =
 	"  --force-panel-type             force gamescope to treat the display as either internal or external\n"
 	"  --force-windows-fullscreen     force windows inside of gamescope to be the size of the nested display (fullscreen)\n"
 	"  --cursor-scale-height          if specified, sets a base output height to linearly scale the cursor against.\n"
+	"  --lease-connector              if specified, marks a display connector for DRM leasing.\n"
+	"  --ignore-touch-device          if specified, disables touch input for a given display.\n"
 	"  --virtual-connector-strategy   Specifies how we should make virtual connectors.\n"
 	"  --hdr-enabled                  enable HDR output (needs Gamescope WSI layer enabled for support from clients)\n"
 	"                                 If this is not set, and there is a HDR client, it will be tonemapped SDR.\n"
